.github/workflows/tests.yml

```diff
@@ -186,17 +186,39 @@ jobs:
           brew test-bot
         fi
 
-    - name: Deploy the latest Docker image
+    - name: Deploy the latest Docker image to GitHub
       if: matrix.os == 'ubuntu-latest' && github.ref == 'refs/heads/master'
       run: |
         docker login docker.pkg.github.com -u BrewTestBot -p ${{secrets.GITHUB_TOKEN}}
         docker tag brew docker.pkg.github.com/homebrew/brew/brew
         docker push docker.pkg.github.com/homebrew/brew/brew
 
-    - name: Deploy the tagged Docker image
+    - name: Deploy the tagged Docker image to GitHub
       if: matrix.os == 'ubuntu-latest' && startsWith(github.ref, 'refs/tags/')
       run: |
         docker login docker.pkg.github.com -u BrewTestBot -p ${{secrets.GITHUB_TOKEN}}
         v=${GITHUB_REF:10}
         docker tag brew "docker.pkg.github.com/homebrew/brew/brew:$v"
         docker push "docker.pkg.github.com/homebrew/brew/brew:$v"
+
+  docker:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version: ["18.04", "20.04"]
+    if: startsWith(github.ref, 'refs/tags/')
```

Comment:
When I put the `if` here, the pull request UI displays `GitHub Actions CI / docker (pull_request) Queued — Waiting to run this check...` indefinitely, which is harmless, but also annoying. Alternatively, I can put an `if` before each `run` below. @MikeMcQuaid Thoughts?

Reply:
Yeh, it's a bit annoying, I agree. I think what Actions may be nudging us to do is to pull this into a separate workflow file entirely which is only run on published releases.

Reply:
I've created `.github/workflows/docker.yml`